external-import/crowdstrike/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
       - CROWDSTRIKE_REPORT_STATUS=New
       - CROWDSTRIKE_REPORT_INCLUDE_TYPES=notice,tipper,intelligence report,periodic report
       - CROWDSTRIKE_REPORT_TYPE=threat-report
+      - CROWDSTRIKE_REPORT_TARGET_INDUSTRIES= # Can be used to retrieve only reports related to a sector
```

Comment:
Can you update the README file to add this parameter?

Reply:
I did it ✅